packages/allure-playwright/src/index.ts

```diff
@@ -168,6 +168,13 @@ class AllureReporter implements Reporter {
       );
     }
     allureTest.endTest();
+    const atLeastAllureAttachment = result.attachments.find(a => a.name === "environment.json");
+    if (atLeastAllureAttachment) {
+      const envInfo = atLeastAllureAttachment.body!.toString("utf8");
+      runtime.writeEnvironmentInfo(JSON.parse(envInfo));
+    } else {
+      runtime.writeEnvironmentInfo(undefined);
```

Comment:
why write empty?

Reply:
because if no env variables - allure will use process.env by default
https://github.com/allure-framework/allure-js/blob/master/packages/allure-js-commons/src/AllureRuntime.ts#L59

Reply:
the essential thing here - is attaching environment info for run

Reply:
I don't see why we need unconditionally store all the process.env for each and every test result. I rather just remove `else` branch completely